src/Numerics/ODESolvers/SplitExplicitMethod.jl

```diff
@@ -0,0 +1,192 @@
+  \\dot{Q_fast} = f_fast(Q_fast, Q_slow, t)
+  \\dot{Q_slow} = f_slow(Q_slow, Q_fast, t)
+```
+
+with the required time step size `dt` and optional initial time `t0`.  This
+time stepping object is intended to be passed to the `solve!` command.
+
+This method performs an operator splitting to timestep the vertical average of the model at a faster rate than the full model. This results in a first-order time stepper.
+
+### References
+"""
+mutable struct SplitExplicitSolver{SS, FS, RT} <: AbstractODESolver
+    "slow solver"
+    slow_solver::SS
+    "fast solver"
+    fast_solver::FS
+    "time step"
+    dt::RT
+    "time"
+    t::RT
+    "param to couple the models"
+    coupled::Bool
+    "param to step past slow dt"
+    add_fast_substeps::RT
+
+    function SplitExplicitSolver(
+        slow_solver::LSRK2N,
+        fast_solver,
+        Q = nothing;
+        dt = getdt(slow_solver),
+        t0 = slow_solver.t,
+        coupled = true,
+        add_fast_substeps = 0,
+    ) where {AT <: AbstractArray}
+        SS = typeof(slow_solver)
+        FS = typeof(fast_solver)
+        RT = real(eltype(slow_solver.dQ))
+        return new{SS, FS, RT}(
+            slow_solver,
+            fast_solver,
+            RT(dt),
+            RT(t0),
+            coupled,
+            RT(add_fast_substeps),
+        )
+    end
+end
+
+function dostep!(
+    Qvec,
+    split::SplitExplicitSolver{SS},
+    param,
+    time::Real,
+) where {SS <: LSRK2N}
+    slow = split.slow_solver
+    fast = split.fast_solver
+
+    Qslow = Qvec.slow
+    Qfast = Qvec.fast
+
+    dQslow = slow.dQ
+    dQ2fast = similar(dQslow)
+
+    slow_bl = slow.rhs!.balance_law
+    fast_bl = fast.rhs!.balance_law
+
+    groupsize = 256
+
+    slow_dt = getdt(slow)
+    fast_dt_in = getdt(fast)
+
+    for slow_s in 1:length(slow.RKA)
+        # Current slow state time
+        slow_stage_time = time + slow.RKC[slow_s] * slow_dt
+
+        # Initialize fast model and tentency adjustment before evalution of slow mode
+        if split.coupled
+            initialize_states!(
+                slow_bl,
+                fast_bl,
+                slow.rhs!,
+                fast.rhs!,
+                Qslow,
+                Qfast,
+            )
+        end
+
+        # Evaluate the slow mode
+        # --> save tendency for the fast
+        slow.rhs!(dQ2fast, Qslow, param, slow_stage_time, increment = false)
+
+        # vertically integrate slow tendency to advance fast equation
+        # and use vertical mean for slow model (negative source)
+        # ---> work with dQ2fast as input
+        if split.coupled
+            tendency_from_slow_to_fast!(
+                slow_bl,
+                fast_bl,
+                slow.rhs!,
+                fast.rhs!,
+                Qslow,
+                Qfast,
+                dQ2fast,
+            )
+        end
+
+        # Compute (and RK update) slow tendency
+        slow.rhs!(dQslow, Qslow, param, slow_stage_time, increment = true)
```

Comment:
I think you can replace this with something like
```suggestion
        dQslow .+= dQ2fast
```

Reply:
(unless `Qslow` is modified in `tendency_from_slow_to_fast`)

Reply:
@jm-c can you test this change in your branch?